lib/evmone/baseline_instruction_table.cpp

```diff
@@ -0,0 +1,47 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2020 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "baseline_instruction_table.hpp"
+#include "instruction_traits.hpp"
+
+namespace evmone
+{
+namespace
+{
+template <evmc_revision Rev>
+constexpr InstructionTable create_instruction_table() noexcept
+{
+    InstructionTable table{};
+    for (size_t i = 0; i < table.size(); ++i)
+    {
+        auto& t = table[i];
+        const auto gas_cost = instr::gas_costs<Rev>[i];
+        t.gas_cost = gas_cost;  // Include instr::undefined in the table.
+        t.stack_height_required = instr::traits[i].stack_height_required;
+        t.can_overflow_stack = instr::traits[i].stack_height_change > 0;
```

Comment:
Could add
```
static_assert(instr::traits[i].stack_height_change <= 1);
```
because the stack overflow check now assumes this.

Reply:
Done with `assert()`.